src/components/PageHero.tsx

```diff
@@ -117,12 +42,49 @@ const PageHero: React.FC<IProps> = ({
 }) => {
   const { buttons, title, header, subtitle, image, alt } = content
   return (
-    <Content>
-      <HeroContainer isReverse={isReverse} className={className}>
-        <HeroContent>
-          <Title>{title}</Title>
-          <Header>{header}</Header>
-          <Subtitle>{subtitle}</Subtitle>
+    <Box py={4} px={8} width="full">
+      <Flex
+        justifyContent="space-between"
+        mt={8}
+        px={{ base: 0, lg: 16 }}
+        direction={{ base: isReverse ? "column" : "column-reverse", lg: "row" }}
+        className={className}
+      >
+        <Box
+          maxW={{ base: "full", lg: "container.sm" }}
+          py={{ base: 16, lg: 32 }}
+          pl={{ base: 0, lg: 8 }}
+          mr={4}
+        >
+          <Heading
+            as="h1"
+            textTransform="uppercase"
+            fontSize="md"
+            fontWeight="normal"
+            mb={4}
+            color="text300"
+          >
+            {title}
+          </Heading>
+          <Heading
+            as="h2"
+            fontWeight="bold"
+            fontSize={{ base: "2.5rem", lg: "5xl" }}
+            maxW="full"
+            mb={0}
+            color="text00"
+          >
```

Comment:
needs explicit `lineHeight`
```suggestion
          <Heading
            as="h2"
            fontWeight="bold"
            fontSize={{ base: "2.5rem", lg: "5xl" }}
            maxW="full"
            mb={0}
            color="text00"
            lineHeight={1.4}
          >
```